resources/authgear/templates/en/web/authflowv2/__settings_item.html

```diff
@@ -0,0 +1,52 @@
+<!-- Example usage:
+{{ template "authflowv2/__settings_item.html"
+  (dict
+    "Label" "email"
+    "RedirectURL" "url"
+    "Icon" `email`
+    "IdentityArr" $.Emails
+    "Identity" $.Email
```

Comment:
This settings-item is not only for `Identity`. Please rename the context variable name. Also, a single variable for contents is enough because you can deduce "`Identity`" from "`IdentityArr`"